guide/src/class.md

```diff
@@ -271,7 +271,7 @@ You can also inherit native types such as `PyDict`, if they implement
 However, because of some technical problems, we don't currently provide safe upcasting methods for types
 that inherit native types. Even in such cases, you can unsafely get a base class by raw pointer conversion.
 
-```rust
+```rust,no_run
```

Comment:
What is the problem with this?

Reply:
subclasses a builtin type which isn't support on abi3 (and I don't know how to make a doc test `no_run` only in some cfgs)

Reply:
OK, though I'll have to modify this when it is changed to raise a compile error.